pr_agent/algo/utils.py

```diff
@@ -61,3 +64,25 @@ def parse_code_suggestion(code_suggestions: dict) -> str:
     markdown_text += "\n"
     return markdown_text
 
+
+def try_fix_json(review, max_iter=10):
```

Comment:
Consider adding a comment explaining the purpose of the `iter_count` variable and how it contributes to preventing infinite loops. This will make the code more readable and maintainable. [medium]